### DISCUSSION TITLE
Question about fine tune.

### DISCUSSION THREAD

**Discussion by JunleiYang** [unitaryai/detoxify#30] [Q&A]
Hello, there
Currently, I have a task to classify texts to two classes, blocked and approved. This task is quite like [Toxic Comment Classification Challenge](https://www.kaggle.com/c/jigsaw-toxic-comment-classification-challenge). But there are some slight different with this data set. Can I train my own data with the pre-trained model? 

Thanks a lot.

**Comment by laurahanu** [comment]
Hello!

Yes, should work, would probably have to load the detoxify pre-trained model weights and change the final layer so it predicts only 2 classes and just use binary cross_entropy as a loss. Should also change the sigmoid activation function to a softmax.

Hope this helps!
